packages/waku/src/config.ts

```diff
@@ -2,6 +2,10 @@ import type { UserConfig } from 'vite';
 import type { Middleware } from './lib/middleware/types.js';
 
 export type { Middleware };
+export type BuiltinMiddleware =
```

Comment:
On second thought, I don't think we need to export this type. What do you think? Can we revert changes in packages/waku/src/lib/config.ts ?